src/modules/task/index.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const taskSchema = new Schema ({
+  text: String,
+  isCheck: Boolean
+});
+
+module.exports =  Taks = mongoose.model('task', taskSchema);
```

Comment:
extra spaces and the model is called with a capital letter and in the plural:
`mongoose.model('Tasks', taskSchema);`